### PR TITLE
fix: [CHAOS-1659]: removing CHAOS_ENABLED feature flag

### PR DESCRIPTION
**Pull request by SahilKr24**
### Summary

Removing CHAOS_ENABLED flag and replacing it with license check.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by SahilKr24** [comment]
retrigger titlecheck

**Comment by SahilKr24** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx [hunk 1]
```diff
@@ -285,7 +284,7 @@ export default function ExecutionTabs(props: ExecutionTabsProps): React.ReactEle
     })
   }
 
-  if ((isCD || isCDInPipeline) && isChaosEnabled) {
+  if (isCD || isCDInPipeline) {
```
Comment (arkajyotiMukherjee) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx:287]: we might need to update this as now chaos is supported in FF stage as well.

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Review by dhruvUpadhyay07 — DISMISSED** [review]

**Review by arkajyotiMukherjee — DISMISSED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx [hunk 2]
```diff
@@ -45,6 +46,13 @@ interface ExecutionTabsProps {
   setSavedExecutionView: (data: string) => void
 }
 
+function isChaosStepInPipeline(data: ExecutionGraph | undefined): boolean {
+  return Object.values(data?.nodeMap ?? {}).some(entry => {
+    if (entry.stepType === 'Chaos') return true
+    return false
+  })
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx:53]:
```suggestion
  return Object.values(data?.nodeMap ?? {}).some(entry => {
    return entry.stepType === 'Chaos'
  })
```

**Review by rameshx — APPROVED** [review]

**Review by hrishavjha — APPROVED** [review]

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Pull request merged**